pages/packages/helm.md

```diff
@@ -0,0 +1,116 @@
+# Helm
+
+Buildkite Packages provides Helm registry support for distributing Helm charts.
+
+This page is for standard helm publishing instructions, alternatively you can also publish to an [OCI-based registry](/docs/packages/helm-oci).
+
+Once your Helm registry has been [created](/docs/packages/manage-registries#create-a-registry), you can publish/upload charts (generated from `helm package` to create the package) to this registry via the relevant `curl` command presented on your Helm registry's details page.
+
+To view and copy this `curl` command:
+
+1. Select **Packages** in the global navigation to access the **Registries** page.
+1. Select your Helm registry on this page.
+1. Select **Publish a Helm Chart** and in the resulting dialog, use the copy icon at the top-right of the code box to copy this `curl` command and run it to publish your chart to your Helm registry.
+
+This command provides:
+
+- The specific URL to publish a package to your specific Helm registry in Buildkite.
+- The API access token required to publish packages to your Helm registry (if private registry).
```

Comment:
```suggestion
- The API access token required to publish packages to your Helm registry.
```

Unless no access token is needed to publish to a public Helm registry?

Reply:
heh good catch